docs/hub/models-widgets.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Model Widgets
+---
+
+<h1>Widgets</h1>
+
+## What's a widget?
+
+Many model repos have a widget that allows anyone to run inferences directly in the browser!
+
+Here are some examples:
+* [Named Entity Recognition](https://huggingface.co/spacy/en_core_web_sm?text=My+name+is+Sarah+and+I+live+in+London) using [spaCy](https://spacy.io/).
+* [Image Classification](https://huggingface.co/google/vit-base-patch16-224) using [🤗 Transformers](https://github.com/huggingface/transformers)
+* [Text to Speech](https://huggingface.co/julien-c/ljspeech_tts_train_tacotron2_raw_phn_tacotron_g2p_en_no_space_train) using [ESPnet](https://github.com/espnet/espnet).
+* [Sentence Similarity](https://huggingface.co/osanseviero/full-sentence-distillroberta3) using [Sentence Transformers](https://github.com/UKPLab/sentence-transformers).
+
+You can try out all the widgets [here](https://huggingface-widgets.netlify.app/).
+
+## Creating a Widget
+
+A widget is automatically created for your model when you upload it to the Hub. To determine which pipeline and widget to display (`text-classification`, `token-classification`, `translation`, etc.), we analyze information in the repo, such as the metadata provided in the model card and configuration files. This information is mapped to a single `pipeline_tag`. We choose to expose **only one** widget per model for simplicity.
```

Comment:
Maybe link to the model-cards page so users can see where this is all controlled?